paas/web/manifest-dev.yml

```diff
@@ -1,8 +1,8 @@
 ---
 applications:
   - name: teaching-vacancies-dev
-    buildpacks:
-      - ruby_buildpack
+    docker:
```

Comment:
I suggest to leave dev as it is for now until we show the team members how to deploy manually with docker